web3js-ext/src/web3/account.ts

```diff
@@ -1,18 +1,46 @@
+import Web3, {Web3Context} from "web3";
 import {
 	TransactionSigningError,
+	UndefinedRawTransactionError,
 } from 'web3-errors';
 import {
+	Bytes,
 	HexString,
+	Address,
 } from 'web3-types';
 import {
 	bytesToHex,
 	hexToBytes,
 	sha3Raw,
+	toChecksumAddress,
 } from 'web3-utils';
 
-import { TypedTransaction, SignTransactionResult } from "web3-eth-accounts";
+import { 
+	Web3Account,
+	Transaction,
+	TransactionFactory,
+	TypedTransaction, 
+	sign,
+	signTransaction,
+	SignTransactionResult, 
+	parseAndValidatePrivateKey,
+	privateKeyToAddress,
+	privateKeyToAccount,
+	encrypt,
+} from "web3-eth-accounts";
 import { isNullish } from 'web3-validator';
 
+import { prepareTransaction } from "./klaytn_tx";
+
+// TODO: Change the path after web3-core deployed
+const { objectFromRLP } = require("../../../../ethers-ext/dist/src");
+
+import { KlaytnTxFactory} from "@klaytn/ethers-ext";
+
+// eslint-disable-next-line import/extensions
+import * as ethereumCryptography from 'ethereum-cryptography/secp256k1.js';
+
+const secp256k1 = ethereumCryptography.secp256k1 ?? ethereumCryptography;
```

Comment:
```js
export const signTransactionWithContext = async(...) {
  let tx = await prepareTransaction(transaction, context, privateKey);
  let priv = bytesToHex(privateKey);
  return signTransaction(tx, priv);
}
```
And then update `accounts_signTransaction`

Reply:
signTransactionWithContext is moved into initAccountsForContext in https://github.com/klaytn/web3klaytn/pull/522/commits/5d13af491f3abb182a3c7bbf23833af9ac746ba7.